carWarGame.py

```diff
@@ -51,14 +52,19 @@ def shuffle(self) -> None:
         """Shuffles the deck of cards"""
         random.shuffle(self.cards)
 
+    def split(self) -> tuple:
```

Comment:
This is only for 2 players? or might it be in future for 2 + players? it would be good idea include num_players parameter?

Reply:
Now implementing from 1 to 4 players.